apps/condo/domains/billing/schema/BillingIntegrationOrganizationContext.js

```diff
@@ -13,6 +13,13 @@ const { hasValidJsonStructure } = require('@condo/domains/common/utils/validatio
 // TODO(pahaz): move if after Organization refactoring
 const { ORGANIZATION_OWNED_FIELD } = require('../../../schema/_common')
 
+// WARNING!
+// Changing this Array will resolve to creating mutation
+const BillingIntegrationOrganizationContextStatuses = [
```

Comment:
Use the same style like here: https://github.com/open-condo-software/condo/blob/master/apps/condo/domains/notification/constants.js#L61-L70

Reply:
resolved 👍